src/main/Extensions/EmptyTrash/EmptyTrashExtension.ts

```diff
@@ -0,0 +1,116 @@
+import type { AssetPathResolver } from "@Core/AssetPathResolver";
+import type { Extension } from "@Core/Extension";
+import { type FluentIcon, type SearchResultItem } from "@common/Core";
+import type { Image } from "@common/Core/Image";
+
+type Settings = {
+    confirmBeforeEmpty: boolean;
+};
```

Comment:
As stated in the previous comment, there is no UI to adjust these settings so I suggest removing this type as it's not needed.